sourcegraph/sourcegraph
# App: Support deep link launch on non-mac and move funcs to common lib file

Add support for launching with scheme urls (deep links) on non-Mac platforms.

Add a missing `show_window` call for scheme urls (now that the app keeps running "in the tray", we need to show and focus the window when opening a deep link)

Move common functions into `common.rs`.

To do before merging:

- [ ] Test a linux build

## Test plan

- build app
- check that deep links work
- check that everything else works as expected
